pkg/server/registry/images/strategy.go

```diff
@@ -59,35 +59,6 @@ func newStorageStrategy(kclient kclient.WithWatch) (strategy.CompleteStrategy, e
 
 // TODO migrate the logic to validateUpdate when create is removed
 func (s *Strategy) Validate(ctx context.Context, obj runtime.Object) (result field.ErrorList) {
-	image := obj.(*apiv1.Image)
-	duplicateTag := make(map[string]bool)
-
-	for _, tag := range image.Tags {
```

Comment:
On update we should still validate that the new tag list is unique.  Is that check still enforced?

Reply:
the only call that goes straight to update is through a deletion, now all calls route through the sub resource image/tags where the unique tag validation is now happening is that sufficient?

Reply:
I added it back & changed the order in which the image/tags runs. It will now update all duplicate tags and then either create the new image or update the existing one w/ the tags